WooCommerce/Classes/ViewRelated/Fancy Alerts/FancyAlertViewController+SwitchStore.swift

```diff
@@ -0,0 +1,118 @@
+import WordPressUI
+import Yosemite
+
+/// Encapsulates the logic that decides when the fancy alert informing that users can switch stores is presented.
+/// The fancy alert will be presented only once.
+///
+final class SwitchStoreAlertLauncher {
```

Comment:
Picky - I see why you put the `SwitchStoreAlertLauncher` class inside of the fancy alert extension file, but I would argue that this should be its own file. @bummytime having a separate file has been a style convention for woo, but it's never been formalized. Thoughts?

Reply:
My thoughts are that it's OK everything lives in the same file since it's all related to the same fancy alert (and will probably be removed after a few releases anyway 😄 ).

Reply:
>it's OK everything lives in the same file since it's all related
That's fair. You are generating some great discussions @ctarda :)

Reply:
😊